test/test_torch.py

```diff
@@ -9705,6 +9705,12 @@ def test_contains(self):
         val += 10
         self.assertEqual(val in x, False)
 
+        with self.assertRaisesRegex(
+            RuntimeError,
+            'Tensor.__contains__ only supports scalars'):
```

Comment:
It would be nice to check the second part of the message, the "you passed in a %s". Could you add it to the error message? 

'Tensor.__contains__ only supports scalars, but you passed in a str'

Reply:
okay